### PR TITLE
fix: past elections

### PR DESCRIPTION
**Pull request by GoldWolf115**
Fix past elections

- Past Elections: Total staked value does not include vote stakes #4090

### CODE REVIEW

**Review by chrlschwb — APPROVED** [review]
ok

File: packages/ui/src/app/pages/Election/PastElections/PastElections.tsx [hunk 1]
```diff
@@ -65,7 +66,7 @@ export const PastElections = () => {
   return <PageLayout header={header} main={displayMain()} />
 }
 
-export const PastElectionsColLayout = '48px 176px 156px 100px 100px'
+export const PastElectionsColLayout = '48px 100px 100px 100px 80px 80px'
```
Comment (thesan) [inline packages/ui/src/app/pages/Election/PastElections/PastElections.tsx:69]: There's enough space even on 1280px screen to keep the column headers on one line:
```suggestion
export const PastElectionsColLayout = '48px 176px 140px 140px 100px 100px'
```
image

File: packages/ui/src/council/queries/__generated__/council.generated.tsx [hunk 2]
```diff
@@ -394,7 +394,7 @@ export type PastElectionRoundFieldsFragment = {
   endedAtTime?: any | null
   endedAtNetwork?: Types.Network | null
   candidates: Array<{ __typename: 'Candidate'; stake: string }>
-  castVotes: Array<{ __typename: 'CastVote'; voteForId?: string | null }>
+  castVotes: Array<{ __typename: 'CastVote'; voteForId?: string | null; stake: string }>
```
Comment (thesan) [inline packages/ui/src/council/queries/__generated__/council.generated.tsx:397]: You shouldn't update the generated files directly (ideally these shouldn't be tracked anymore).

Instead please edit `packages/ui/src/council/queries/council.graphql`, then run `yarn workspace @joystream/pioneer queries:generate`

File: packages/ui/src/council/components/election/pastElection/PastElectionsList/PastElectionsListRow.tsx [hunk 3]
```diff
@@ -30,6 +29,7 @@ export const PastElectionsListRow = ({ election }: PastElectionsListRowProps) =>
         <></>
       )}
       <TokenValue value={election.totalStake} />
```
Comment (thesan) [inline packages/ui/src/council/components/election/pastElection/PastElectionsList/PastElectionsListRow.tsx:31]: To clarify the code please rename this field:
```suggestion
      <TokenValue value={election.totalCandidatesStake} />
```

Here and in `packages/ui/src/council/types/PastElection.ts` too.

**Review by thesan — CHANGES_REQUESTED** [review]
Thank you for the PR.

To fix the CI you should:
1. Fix the code linting by running: `yarn lint:fix`
2. Fix the interaction test by pulling the `dev` branch (because the storybook deployed from this branch is missing some new stories recently merged on `dev`).

**Comment by thesan** [comment]
@GoldWolf115 sorry I just realized that the interaction test failed earlier due to the GH workflow for new contributor (currently I should think of relaunching the storybook Vercel deployment before approving the GH CI run). However there's no harm in pulling the `dev` branch.

But now you need to rename `totalStake` to `totalCandidatesStake` in `@/council/components/election/pastElection/PastElectionTabs.tsx` because the Vercel deployments are failing ATM: https://vercel.com/joystream/dao/Hhx5dG8L4cyArcxMzjZzbMqz6Dxr?filter=all. To avoid these kind issues try to remember to check if the app is running locally with `yarn start` before pushing new commits.

**Pull request closed without merging**